src/stactools/lila_hkh_glacier/commands.py

```diff
@@ -0,0 +1,182 @@
+import click
+import os
+import logging
+
+from stactools.lila_hkh_glacier import cog
+from stactools.lila_hkh_glacier import stac
+from stactools.lila_hkh_glacier import utils
+
+logger = logging.getLogger(__name__)
+
+
+def create_lilahkhglacier_command(cli):
+    """Creates the lilahkhglacier command line utility."""
+    @cli.group(
+        "lilahkhglacier",
+        short_help=(
+            """Commands for working with LILA HKH Glacier Mapping data"""),
+    )
+    def lilahkhglacier():
+        pass
+
+    @lilahkhglacier.group(
+        "lilahkhglacier-slice",
```

Comment:
Is the `lilahkhglacier-` prefix needed? `stac lilahkhglacier lilahkhglacier-slice ...` is a mouthful

Reply:
I agree, but the downstream process that I need to use this in matches the name of the command (`lilahkhglacier-slice`) to the name of the collection into which the item is placed. Are you aware of any other stactools packages that create multiple collections that I could look at?